cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1028,7 +1028,8 @@ private boolean isInterBrokerReplicaActionDone(Cluster cluster, TopicPartition t
       Node[] currentOrderedReplicas = cluster.partition(tp).replicas();
       switch (task.state()) {
         case IN_PROGRESS:
-          return task.proposal().isInterBrokerMovementCompleted(currentOrderedReplicas);
+          return task.proposal().isInterBrokerMovementCompleted(currentOrderedReplicas)
```

Comment:
`ExecutionProposal#isInterBrokerMovementCompleted` is called by other functions, as well. 

Does the same update not apply them? If it does, should we move `&& currentOrderedReplicas.length == cluster.partition(tp).inSyncReplicas().length;` in this function? If not, should we rename this function to something else to make it more clear?

Reply:
Good point! I would like to apply the logic to all the place so we better honor the concurrency throttling.